db/seeds.rb

```diff
@@ -0,0 +1,53 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+#   Character.create(name: 'Luke', movie: movies.first)
+
+
+puts "Destroying old records"
+
+Person.destroy_all
+Following.destroy_all
+
+puts "Old records destroyed"
+
+
+MULTIPLIER = 10
+
+
+def generate_person
+  u = Person.new
+  u[:name]  = Faker::Name.first_name
+  u[:age]   = rand(18..60)
+  u[:occupation] = Faker::Job.title
+  u.save
+end
+
+
+(MULTIPLIER*50).times { generate_person }
+puts "Created persons"
+
+@test_persons = []
+Person.all.each {|u| @test_persons << u}
+@test_persons.shuffle!
+
+
+def generate_join_table
+  f = Following.new
+  person_idx = @test_persons.pop
+  f.acceptor_id = person_idx.id
```

Comment:
to generate just a list of ids only, you can use `Person.pluck(:id)` which will return an array of ids from `Person`